### ISSUE TITLE
"Could not find any image file" Error when running test.lua

### ISSUE THREAD

**Issue by larry-he** [junyanz/CycleGAN#37]
I downloaded the horse2zebra dataset and the horse2zebra pretrained model.

Running "DATA_ROOT=./datasets/horse2zebra name=horse2zebra_pretrained phase=test th test.lua" in the CycleGAN directory gave the below output. I appreciate any help I can get solving this.

------------------- Options -------------------	
                DATA_ROOT: ./datasets/horse2zebra	
               align_data: 0	
             aspect_ratio: 1	
                batchSize: 1	
          checkpoints_dir: ./checkpoints	
           continue_train: 1	
                    cudnn: 1	
                  display: 1	
               display_id: 200	
                 fineSize: 128	
                     flip: 0	
                      gpu: 1	
                 how_many: all	
                 input_nc: 3	
                 loadSize: 128	
                    model: cycle_gan	
                 nThreads: 1	
                     name: horse2zebra_pretrained	
                     norm: instance	
                output_nc: 3	
                    phase: test	
               preprocess: regular	
           resize_or_crop: resize_and_crop	
              results_dir: ./results/	
           serial_batches: 1	
                     test: 1	
          which_direction: AtoB	
              which_epoch: latest	
-----------------------------------------------	
GPU Mode	
DataLoader UnalignedDataLoader was created.	
Starting donkey with id: 1 seed: 8785
table: 0x40ff0d18
table: 0x40fdfb48
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
now combine all the files to a single large file
load the large concatenated list of sample paths to self.imagePath
cmd..wc -L '/tmp/lua_SzUjGM' |cut -f1 -d' '
/home/larry/torch/install/bin/luajit: /home/larry/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 1 callback] /home/larry/Desktop/CycleGAN/data/dataset.lua:216: Could not find any image file in the given input paths
stack traceback:
	[C]: in function 'assert'
	/home/larry/Desktop/CycleGAN/data/dataset.lua:216: in function '__init'
	/home/larry/torch/install/share/lua/5.1/torch/init.lua:91: in function </home/larry/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'dataLoader'
	/home/larry/Desktop/CycleGAN/data/donkey_folder.lua:205: in main chunk
	[C]: in function 'dofile'
	/home/larry/Desktop/CycleGAN/data/data.lua:39: in function </home/larry/Desktop/CycleGAN/data/data.lua:29>
	[C]: in function 'xpcall'
	/home/larry/torch/install/share/lua/5.1/threads/threads.lua:234: in function 'callback'
	/home/larry/torch/install/share/lua/5.1/threads/queue.lua:65: in function </home/larry/torch/install/share/lua/5.1/threads/queue.lua:41>
	[C]: in function 'pcall'
	/home/larry/torch/install/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
	[string "  local Queue = require 'threads.queue'..."]:13: in main chunk
stack traceback:
	[C]: in function 'error'
	/home/larry/torch/install/share/lua/5.1/threads/threads.lua:183: in function 'dojob'
	/home/larry/torch/install/share/lua/5.1/threads/threads.lua:264: in function 'synchronize'
	/home/larry/torch/install/share/lua/5.1/threads/threads.lua:142: in function 'specific'
	/home/larry/torch/install/share/lua/5.1/threads/threads.lua:125: in function 'Threads'
	/home/larry/Desktop/CycleGAN/data/data.lua:27: in function 'new'
	/home/larry/Desktop/CycleGAN/data/data_util.lua:17: in function 'load_dataset'
	./data/unaligned_data_loader.lua:24: in function 'Initialize'
	test.lua:47: in main chunk
	[C]: in function 'dofile'
	...lice/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405d50

**Comment by aferriss** [comment]
I actually came across this error too and strangely taking out the phase=test entirely from the initial opt seemed to solve it. Not entirely sure why...

**Comment by junyanz** [comment]
Could you try the following command? 
```bash
DATA_ROOT=./datasets/horse2zebra name=horse2zebra_pretrained model=one_direction_test phase=test loadSize=256 fineSize=256 resize_or_crop="scale_width" th test.lua
```

**Comment by larry-he** [comment]
Updating all dependencies and getting the latest version of this repo fixed everything.
